commands.js

```diff
@@ -1563,7 +1563,7 @@ exports.commands = {
 			this.add("|raw|<div class=\"broadcast-red\"><b>Moderated chat was set to " + modchat + "!</b><br />Only users of rank " + modchat + " and higher can talk.</div>");
 		}
 		if (room.battle && !room.modchat && !user.can('modchat')) room.requestModchat(null);
-		this.logModCommand(user.name + " set modchat to " + room.modchat);
+		this.privateModCommand(user.name + " set modchat to " + room.modchat);
```

Comment:
Generally these are surrounded by outer parenthesis, ex: ``(user set modchat to +)`` is what would be displayed in the room.